node/src/manager/commands/config.rs

```diff
@@ -16,10 +16,41 @@ use graph::{
     slog::Logger,
 };
 use graph_chain_ethereum::NodeCapabilities;
-use graph_store_postgres::DeploymentPlacer;
+use graph_store_postgres::{BlockStore, DeploymentPlacer};
 
 use crate::{config::Config, network_setup::Networks};
 
+/// Compare the NetIdentifier of all defined adapters with the existing
+/// identifiers on the ChainStore. If a ChainStore doesn't exist it will be show
+/// as an error. It's intended to be run again an environment that has already
+/// been setup by graph-node.
+pub async fn providers(networks: &Networks, store: Arc<BlockStore>) -> Result<(), Error> {
+    println!("Checking providers");
+    for (chain_id, ids) in networks.all_chain_identifiers().await.into_iter() {
+        let (_oks, errs): (Vec<_>, Vec<_>) = ids
+            .into_iter()
+            .map(|(provider, id)| {
+                id.map_err(IdentValidatorError::from)
+                    .and_then(|id| store.check_ident(chain_id, &id).map(|_| (provider, id)))
+            })
+            .partition_result();
+        let errs = errs
+            .into_iter()
+            .dedup()
+            .collect::<Vec<IdentValidatorError>>();
+
+        if errs.is_empty() {
+            println!("chain_id: {}: status: OK", chain_id);
+            continue;
+        }
+
+        println!("chain_id: {}: status: NOK", chain_id);
+        println!("errors: {:?}", errs);
```

Comment:
I think it would be helpful to printout the provider label of the specific provider which failed the test. Not sure if it already does that, the errors seemed to be only printing the chain name